cfgov/v1/models/base.py

```diff
@@ -131,6 +131,14 @@ class CFGOVPage(Page):
         ObjectList(settings_panels, heading='Configuration'),
     ])
 
+    def get_authors(self):
+        """ Returns a sorted list of authors. Default is alphabetical """
+        return self.alphabetize_authors()
+
+    def alphabetize_authors(self):
+        """ Alphabetize authors of this page by last name """ 
+        return sorted(self.authors.names(), key=lambda x: x.split(" ")[-1])
```

Comment:
Interesting. Apparently the behavior [isn't quite identical](https://docs.python.org/2/library/stdtypes.html#str.split). Just using `x.split()` would correctly handle a case where someone accidentally puts an extra space at the end:

```py
>>> 'Donald Duck '.split()[-1]
'Duck'
>>> 'Donald Duck '.split(' ')[-1]
''
```

Although, it appears that Taggit correctly strips any extra spaces (try entering `" Donald  Duck "` into the authors list and you'll see it correctly converts it to `"Donald Duck"`.